src/strings/en-us.json

```diff
@@ -543,8 +543,8 @@
     "LabelAllowedRemoteAddresses": "Remote IP address filter",
     "LabelAllowedRemoteAddressesMode": "Remote IP address filter mode",
     "LabelAllowHWTranscoding": "Allow hardware transcoding",
-    "LabelAppName": "App name",
-    "LabelAppNameExample": "Example: Sickbeard, Sonarr",
+    "LabelAppName": "Human Readable Name",
```

Comment:
I think leaving this as "App name" would be fine with the improved example/help text. 🤷‍♂️

Reply:
Maybe should also be fine